src/ddmd/root/rmem.d

```diff
@@ -55,6 +55,11 @@ version (GC)
         }
     }
 
+    extern (C) void* allocmemory(size_t m_size) nothrow
+    {
+        return GC.malloc(m_size);
+    }
```

Comment:
It seems like `version(GC)` isn't used actively (at least by anything that is part of the testsuite/CIs).